python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -1,5 +1,12 @@
 import PromptContainer from "./prompt/PromptContainer";
-import { Container, Group, Button, createStyles, Stack } from "@mantine/core";
+import {
+  Container,
+  Group,
+  Button,
+  createStyles,
+  Stack,
+  Flex,
+} from "@mantine/core";
```

Comment:
Based on my understanding from your response in https://github.com/lastmile-ai/aiconfig/pull/640#issuecomment-1870483897, this is just fancy colors around base components right?

Reply:
For mantine in general? Think of it more like XDS - it's a component library that gives a lot of out-of-the-box functionality and common styles that can be easily updated across the components